homeassistant/components/fastdotcom/__init__.py

```diff
@@ -28,13 +29,21 @@
 DEFAULT_INTERVAL = timedelta(hours=1)
 
 CONFIG_SCHEMA = vol.Schema({
-    DOMAIN: vol.Schema({
-        vol.Optional(CONF_UPDATE_INTERVAL, default=DEFAULT_INTERVAL):
-            vol.All(
-                cv.time_period, cv.positive_timedelta
-            ),
-        vol.Optional(CONF_MANUAL, default=False): cv.boolean,
-    })
+    DOMAIN: vol.All(
+        vol.Schema({
+            vol.Optional(CONF_UPDATE_INTERVAL):
```

Comment:
Shouldn't we have a default here? I don't think we should rely on cv.deprecated to provide the default for the non deprecated option. After deprecated period is over, we should just be able to remove the deprecation validator and not change anything else, right?

Reply:
I think we'll have to make some changes to the deprecation validator then. The way it's currently written, it will always copy from key to replacement_key if replacement_key doesn't exist. By providing a default value, replacement_key will always exist and therefore it will not make that copy.

I think it's easier to manage with the deprecation validator handling the default and then we move the default back to it's key when cleaning up, but let me see what it would take to make it work the other way.

Reply:
Okay , looks like we can handle this with a fairly simple change, so I agree, this is the right way to do this. Will update.

Reply:
Btw, I commented on the wrong line here, but you noticed that I think.

I should have commented on the `CONF_SCAN_INTERVAL` line.